src/sentry/utils/prompts.py

```diff
@@ -4,6 +4,7 @@
     "alert_stream": {"required_fields": ["organization_id"]},
     "sdk_updates": {"required_fields": ["organization_id"]},
     "suggest_mobile_project": {"required_fields": ["organization_id"]},
+    "stacktrace_link": {"required_fields": ["organization_id", "project_id"]},
```

Comment:
@MeredithAnya Does this mean a user who dismisses the prompt on one project will see it for another?

Reply:
That was my intention

Reply:
@MeredithAnya works for me unless we see users start complaining about it 😁

Reply:
@scefali yeah we can always adjust, especially when it goes to GA, but I think it makes more sense on a project level for now